### DISCUSSION TITLE
Running IntelOwl Locally

### DISCUSSION THREAD

**Discussion by sercuz** [intelowlproject/IntelOwl#1179] [General]
Hi there,
I'm wondering if it is possible to run the project locally (I mean not in docker) to make it easier to work in the backed + frontend. Meanwhile, whenever I do a change on the backend I have to re-build the project to test the changes.

**Comment by mlodic** [comment]
Hey, 

thanks for your interest in the project.

Please check out the official documentation because what you have said is not true: https://intelowl.readthedocs.io/en/latest/Contribute.html#how-to-start-setup-project-and-development-instance

Running outside docker would be to just over complicate things and not simplify them.

**Reply by sercuz** [reply]
Thank you @mlodic .
